wicket-core/src/main/java/org/apache/wicket/markup/head/CssHeaderItem.java

```diff
@@ -365,6 +382,27 @@ protected final void internalRenderCSSReference(Response response, String url, S
 		response.write("\n");
 	}
 
+	/**
+	 * @return CSP nonce
+	 */
+	public String getNonce()
+	{
+		return nonce;
+	}
+
+	/**
+	 * Set the CSP nonce
+	 *
+	 * @param nonce
+	 * @return {@code this} object, for method chaining
+	 */
+	public CssHeaderItem setNonce(String nonce)
+	{
+		Args.notEmpty(nonce, "nonce");
```

Comment:
Why shouldn't it be null? We allow that for other attributes too.

Reply:
It makes sense to remove this assert. @solomax @martin-g do you agree?

Reply:
I would say this argument can be `null`
But it is really weird, to call this method with empty/null argument
I would say it can raise WARN to show possible misuse ...

Reply:
My idea was, that it's a special security attribute, and you don't really want to erase it. So, that's why I wanted it to explode.

Reply:
This is how I understand it:

If CSP is not enabled then this method should not be called at all. 
If CSP is enabled then the parameter should be non-empty.